### PR TITLE
Add locale for Brazilian Portuguese 🇧🇷

### PR DESCRIPTION
**Pull request by cuducos**
Hey you… as I got that new locales are welcomed, and as I just wrote the `pt-BR` locale to use this package [in a project](https://github.com/datasciencebr/jarbas/pull/95), I thought that the PR would be interesting…

Thanks for creating and maintaining _elm-date-extra_ ; )

### CODE REVIEW

**Comment by rluiten** [comment]
Ahh its elm-format, I should have guessed that.

I usually like the case statement formatting of it when I played with it ages back, but for these specific files where the days in a week and months in a year just are not going to change, and where the result is a single string I find the vertical orientation annoying.

I thought elm-format was 2 space indent have the changed again ?
I've been moving to 2 spaces because elm-format was doing that hmm a year ago I suppose now.

What about the the padding code in `dayOfMonthWithSuffix`  is 4 really correct ?

**Comment by rluiten** [comment]
Hmm ok elm-format only does 4, I do remember some discussions on the slack channel and I came away thinking it was changing to 2, I was wrong. :).

File: src/Date/Extra/I18n/I_pt_br.elm [hunk 1]
```diff
@@ -158,16 +86,13 @@ make them same length as double digit days of month.
...
-        else
-            value
+  let
+    value =
+      case day of
+        1 -> "1º"
+        _ -> toString day
+  in
+    if pad then
+      padLeft 4 ' ' value
```
Comment (cuducos) [inline src/Date/Extra/I18n/I_pt_br.elm]: Hum… now I got it. Probably it should be 2 then. My bad. I'll send a PR for that ; )

Comment (rluiten) [inline src/Date/Extra/I18n/I_pt_br.elm]: Don't worry already done.
Ill release it in next few hours with 2 other languages.
I am just writing some test for formatted locale cases as a quick smoke test, should be easy though.

Comment (cuducos) [inline src/Date/Extra/I18n/I_pt_br.elm]: Yay… many thanks  ; )

> Ill release it in next few hours with 2 other languages.

🍾 🎊

**Comment by rluiten** [comment]
Published 8.2.0

**Pull request merged**